src/dndbeyond/base/monster.js

```diff
@@ -283,6 +283,7 @@ class Monster extends CharacterBase {
         if (hit_idx > 0)
             hit = description.slice(hit_idx);
         // Using match with global modifier then map to regular match because RegExp.matchAll isn't available on every browser
+        hit = hit.replace("−", "-");
```

Comment:
use `.replace(/−/g, "-")` so it replaces it all, not just the first one it encounters. Also, should be done on the description itself, not just the to-hit, as it could affect damage value too.